core/ledger/cceventmgmt/mgr.go

```diff
@@ -59,6 +59,28 @@ func (m *Mgr) Register(ledgerid string, l ChaincodeLifecycleEventListener) {
 	m.ccLifecycleListeners[ledgerid] = append(m.ccLifecycleListeners[ledgerid], l)
 }
 
+// RegisterAndInvokeFor registers the listener and in addition invokes the listener for each chaincode that is present in the supplied
+// list of legacyChaincodes and is installed on the peer
+func (m *Mgr) RegisterAndInvokeFor(legacyChaincodes []*ChaincodeDefinition, ledgerid string, l ChaincodeLifecycleEventListener) error {
```

Comment:
A nit, but shouldn't this method just be called `RegisterAndInvoke` or `RegisterAndInvokeListener` ?

Reply:
Was more curious than anything.  As long as we strive for consistency I’m ok